### PR TITLE
run_examples script: Fix run_simple_flow example

### PR DESCRIPTION
**Pull request by nirarg**
This includes the following changes:
* Get the PROJECT_ID and adding it the the Workflow execution
* Change parametes: name to workFlowName & tasks to workFlowTasks
* Add workflow task arguments to restCallTask
* Fix RestAPIWorkFlowTask use the right key for WorkFlowTaskParameters

### CODE REVIEW

File: workflow-examples/run_examples.sh [hunk 1]
```diff
@@ -263,4 +310,5 @@ run_complex_flow() {
   echo "network workflow execution id:" $(echo_green $EXECUTION_ID)
 }
 
-run_complex_flow
+run_simple_flow
+# run_complex_flow
```
Comment (masayag) [inline workflow-examples/run_examples.sh:314]: perhaps make this configurable with 'simple' as default?
e.g. For running the simple flow:
```
workflow-examples/run_examples.sh simple
# or
workflow-examples/run_examples.sh
```
and for complex:
```
workflow-examples/run_examples.sh complex
```
that way there won't be a need to modify the file when using it

Comment (nirarg) [reply workflow-examples/run_examples.sh]: Added argument to control the flow type
Default type is complex, to keep previous behavior

File: workflow-examples/run_examples.sh [hunk 2]
```diff
@@ -263,4 +310,13 @@ run_complex_flow() {
   echo "network workflow execution id:" $(echo_green $EXECUTION_ID)
 }
 
-run_complex_flow
+if [ $# -eq 0 ] || [ $1 = "complex" ]; then
```
Comment (pkliczewski) [inline workflow-examples/run_examples.sh:313]: Do you think it makes sense to add support to run both types sequentially?

Comment (eloycoto) [reply workflow-examples/run_examples.sh:313]: please, do not over complicated when yesterday we agreed to moved this script to another things :-)

Comment (nirarg) [reply workflow-examples/run_examples.sh]: I agree with @eloycoto, I prefer to keep it like this and open tickets for new requirements

**Comment by pkliczewski** [comment]
/lgtm

Please wait with the merge. I would like to merge this [PR](https://github.com/redhat-developer/parodos/pull/93) first.

**Review by lshannon — APPROVED** [review]

**Pull request merged**